src/rail/estimation/algos/deepdisc.py

```diff
@@ -143,11 +145,41 @@ class DeepDiscInformer(CatInformer):
     def __init__(self, args, comm=None):
         CatInformer.__init__(self, args, comm=comm)
 
+    def inform(self, training_data, metadata):
```

Comment:
In the short term, we should rename `metadata` to something like `training_metadata` or `training_set_metadata`.

Reply:
Addressed in #25